closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -857,6 +857,31 @@ private static byte[] intToLittleEndian (long i) {
         return ret;
     }
 
+    private void ChaCha20Init(boolean modeChanged) {
+        // Optimize the initialization of chacha20-poly1305 when they have the same ossl_mode
+        // otherwise, we have to reinitialize based upon the ossl_mode.
+        int ret = nativeCrypto.ChaCha20Init(context, ossl_mode, openssl_iv, openssl_iv.length, keyBytes, keyBytes.length, !modeChanged);
+
+        if (ret == -1) {
+            throw new ProviderException("Error in Native ChaCha20Cipher");
+        }
+        opensslInitialized = true;
+    }
+
+    /**
+     * Reset the Cipher's state to the appropriate values.
```

Comment:
I don't think `Cipher` deserves to be capitalized here.

Reply:
Changed.